src/blocks/stars.css

```diff
@@ -0,0 +1,19 @@
+.stars_star {
+  background: url(../images/star.svg) center no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.card__stars :nth-child(-n+4) {
```

Comment:
add styles for any stars count

Reply:
I don't understand - what should I fixed?

Reply:
i have to be able to change class and choose another amount of stars

Reply:
Like this: ?
.card__stars :nth-child(-n+1),
.card__stars :nth-child(-n+2),
.card__stars :nth-child(-n+3),
.card__stars :nth-child(-n+4),
.card__stars :nth-child(-n+5) {
  background: url(../images/star-active.svg) center no-repeat;
}